ksql-engine/src/main/java/io/confluent/ksql/services/SandboxConnectClient.java

```diff
@@ -32,10 +33,14 @@ private SandboxConnectClient() { }
 
   public static ConnectClient createProxy() {
     return LimitedProxyBuilder.forClass(ConnectClient.class)
-        .swallow("create", methodParams(String.class, Map.class), ConnectResponse.of("sandbox"))
-        .swallow("describe", methodParams(String.class), ConnectResponse.of("sandbox"))
-        .swallow("connectors", methodParams(), ConnectResponse.of(ImmutableList.of()))
-        .swallow("status", methodParams(String.class), ConnectResponse.of("sandbox"))
+        .swallow("create", methodParams(String.class, Map.class),
+            ConnectResponse.of("sandbox", HttpStatus.SC_INTERNAL_SERVER_ERROR))
```

Comment:
What's the point of white-listing these methods if the sandboxed client has them return errors?

Reply:
it's well handled instead of throwing exceptions - i.e. the code doesn't expect "UnsupportedOperationException" it expects some kind of meaningful error response. I don't think it's necessary, but just a little nicer down the line. If you feel strongly that we shouldn't be doing this I can revisit it in another PR.